catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -311,8 +349,12 @@ private String safeGetBody(Request req) {
         return ImmutableMap.of(RESP_MSG, "Successfully updated");
       }
     }
-    catalogFramework.create(new CreateRequestImpl(metacard));
-    return ImmutableMap.of(RESP_MSG, "Successfully created");
+    Metacard createdMetacard =
+        catalogFramework.create(new CreateRequestImpl(metacard)).getCreatedMetacards().get(0);
+    return ImmutableMap.<String, Object>builder()
+        .put(Core.ID, createdMetacard.getId())
+        .put(CREATED_ON, createdMetacard.getAttribute(Core.CREATED).getValue())
+        .build();
```

Comment:
❗️ Now that `PUT` is no longer overloaded to perform the tasks of both create and update, we should enforce proper HTTP semantics and split this method into two. Each can validate things a bit more harshly and fail fast if the request is not well formed, per our expectations. This mostly involves when a caller can provide an ID and when they cannot.